### PR TITLE
feat: UI for article writing #230

### PR DESCRIPTION
**Pull request by nishant0708**
feat: UI for article writing #230

Screenshot 2024-06-21 022134
Screenshot 2024-06-21 022148
Screenshot 2024-06-21 024126
Screenshot 2024-06-21 024236
Screenshot 2024-06-21 024249
Screenshot 2024-06-21 024706

-Navbar is also updated (I tooked ss before)

### CODE REVIEW

**Comment by nishant0708** [comment]
hey @Tanay-ErrorCode @Doraemon012  please Review this pr and let me know changes if any

**Comment by nishant0708** [comment]
Hey @Doraemon012 can you please review this

File: src/components/Article/ArticleWritingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <textarea
+          placeholder="Tell your story..."
+          value={content}
+          onChange={(e) => setContent(e.target.value)}
+          required
+          className="article-textarea"
+        ></textarea>
+      </form>
+      <div className="markdown-preview">
+        <h1>Preview:</h1>
```
Comment (Doraemon012) [inline src/components/Article/ArticleWritingPage.tsx:38]: Remove the preview functionality for now. We'll add it later.

File: src/components/Article/ArticleWritingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const [content, setContent] = useState<string>("");
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    console.log("Article submitted:", { title, content });
+    // Add your form submission logic here
+  };
+
+  return (
+    <div className="article-write">
```
Comment (Doraemon012) [inline src/components/Article/ArticleWritingPage.tsx:16]: Use `MUI`. Make this a card.

**Comment by Doraemon012** [comment]
@nishant0708 
1. Please use `MUI`.
2. Remove the icon and the vertical bar.
image
3. Use [https://mui.com/material-ui/react-text-field/](https://mui.com/material-ui/react-text-field/) for the description and title.
Use [`Multiline Placeholder`](https://mui.com/material-ui/react-text-field/#multiline) for description. And `Single Line Placeholder` for title.


image

**Comment by nishant0708** [comment]
Okay doing it now...

**Comment by nishant0708** [comment]
Hey @Doraemon012 please check it now

**Review by Doraemon012 — APPROVED** [review]

**Pull request merged**